src/uu/sort/src/sort.rs

```diff
@@ -679,9 +702,10 @@ pub fn uumain(args: impl uucore::Args) -> i32 {
         )
         .arg(
             Arg::with_name(OPT_IGNORE_NONPRINTING)
-                .short("-i")
+                .short("i")
                 .long(OPT_IGNORE_NONPRINTING)
-                .help("ignore nonprinting characters"),
+                .help("ignore nonprinting characters")
+                .conflicts_with_all(&[OPT_NUMERIC_SORT, OPT_GENERAL_NUMERIC_SORT, OPT_HUMAN_NUMERIC_SORT, OPT_MONTH_SORT]),
```

Comment:
For some reason rustfmt doesn't pick up this line, even though it is too long....

Reply:
Yeah, this is this bug: https://github.com/rust-lang/rustfmt/issues/3863